data-prepper-core/src/main/java/org/opensearch/dataprepper/acknowledgements/AcknowledgementSetMonitor.java

```diff
@@ -67,55 +55,6 @@ public void add(final AcknowledgementSet acknowledgementSet) {
         }
     }
 
-    public void acquire(final EventHandle eventHandle) {
-        if (eventHandle == null) {
-            numNullHandles.incrementAndGet();
-            return;
-        }
-
-        DefaultAcknowledgementSet acknowledgementSet = getAcknowledgementSet(eventHandle);
-        lock.lock();
-        boolean exists = false;
-        try {
-            exists = acknowledgementSets.contains(acknowledgementSet);
-        } finally {
-            lock.unlock();
-        }
-        // if acknowledgementSet doesn't exist then it means that the
-        // event still active even after the acknowledgement set is
-        // cleaned up.
-        if (exists) {
-            acknowledgementSet.acquire(eventHandle);
-        } else {
-            LOG.warn("Trying acquire an event in an AcknowledgementSet that does not exist");
-            numInvalidAcquires.incrementAndGet();
```

Comment:
All of this was unused?

Reply:
@graytaylor0 It was used but not needed after the new changes where the acquire is directly done on the event handle. When we first implemented this, event handle was not guaranteed to be there, so we have implemented it to go through DefaultAcknowledgementSetManager which in turn invokes AcknowledgementSetMonitor.acquire(). Now all that became unnecessary.